modules/serverless-monitoring-services-default-metrics.adoc

```diff
@@ -0,0 +1,187 @@
+[id="serverless-monitoring-services-default-metrics_{context}"]
+= Knative service metrics exposed by default
+
+The following table lists metrics exposed by default for each Knative service on port 9090.
+
+[options=header]
+|===
+
+|Metric name, unit, and type |Description |Metric tags
+
+// New row
+|`*queue_requests_per_second*`
+
+Metric unit: dimensionless
+
+Metric type: gauge
+|Number of requests per second that hit the queue proxy.
+
+Formula: `stats.RequestCount / r.reportingPeriodSeconds`
+
+Note: `stats.RequestCount` is calculated directly from the networking pkg stats for the given reporting duration.
+|destination_configuration="event-display",
```

Comment:
@adellape There's a certain subtle loss of information if we present these as bullets, plus your note about width is correct. OTOH, while bullet points would've been clearer, currently every key=value entry is shown on a separate line anyway, which helps with reading.
Playing with colwidths doesn't change anything in this case - the columns are restricted, because some of the entries are long.
(Good points, but I can't see how to make them work here; this table is a pain)